doc/over_react_redux_documentation.md

```diff
@@ -277,47 +271,29 @@ set to your additional store.
 
 ## Using Redux DevTools
 Redux DevTools can be set up easily by adding only a few lines of code.
-> These instructions and additional information can be found [here](https://github.com/MichaelMarner/dart-redux-remote-devtools#middleware-configuration)
+> Additional information about `redux_dev_tools` and `DevToolsStore`s can be found [here](https://github.com/brianegan/redux_dev_tools#redux_dev_tools)
 
-1. Add `redux_remote_devtools` and `redux_dev_tools` as dependencies in your `pubspec.yaml`.
+1. Add `redux_dev_tools` as a dev dependency in your `pubspec.yaml`.
     ```yaml
-    dependencies:
+    dev_dependencies:
       redux_dev_tools: 0.4.0
-      redux_remote_devtools: ^0.0.11
     ```
-    > __NOTE:__ The dependency on `redux_remote_devtools` being `>=0.0.11` is a hard line, as prior to this version 
-    the package was tailored to Flutter and reliant on `dart:io`.
-1. Import `redux_remote_devtools` and `redux_dev_tools` into your store file.
+1. Import `redux_dev_tools` into your store file.
     ```dart
-    import 'package:redux_remote_devtools/redux_remote_devtools.dart';
     import 'package:redux_dev_tools/redux_dev_tools.dart';
     ```
-1. Change your store to a `DevToolsStore` instance and add the `remoteDevtools` middleware.
-    ```dart
-    var store = new DevToolsStore<AppState>(
-      /*ReducerName*/, 
-      initialState: /*Default App State Object*/, 
-      middleware: [remoteDevtools],
+1. Change your `Store` to a `DevToolsStore` instance and add the constant `overReactReduxDevToolsMiddleware` to your middleware.
+    ```diff
+    - var store = new Store<AppState>(
```

Comment:
I see the benefit of adding the addition / subtraction signs to show what's being added / removed, but I feel like it makes it ultimately makes the reader's life a little more difficult. Feel free to veto this, but would it suffice to add a sentence to the end of the description that says "The final result should like similar to the example below" and leave the example snippet as is?

Reply:
See my comment below about the diff formatting